tools/dependencies/known-dependencies.txt

```diff
@@ -270,4 +270,12 @@ okhttp-3.14.9.jar
 okio-1.17.2.jar
 jmespath-java-1.12.160.jar
 jackson-dataformat-cbor-2.12.5.jar
-ion-java-1.0.2.jar
\ No newline at end of file
+ion-java-1.0.2.jar
+casdoor-java-sdk-1.5.1.jar
+casdoor-spring-boot-starter-1.2.0.jar
+jcip-annotations-1.0-1.jar
+json-20140107.jar
+lombok-1.18.22.jar
+nimbus-jose-jwt-9.10.1.jar
+org.apache.oltu.oauth2.client-1.0.2.jar
+org.apache.oltu.oauth2.common-1.0.2.jar
```

Comment:
The `lombok` is not belong to `apache 2.0`, so we need to add license in `dolphinscheduler-dist` and the other dependencies belong to `apache 2.0` we still need to announce in NOTICE.

Reply:
@ruanwenjun Do we also need to add licenses of dependencies which belong to `apache 2.0` into `dolphinscheduler-dist`? Because I found a lot of `apache 2.0` licenses in `dolphinscheduler-dist`.

Reply:
you can refer this https://dolphinscheduler.apache.org/en-us/community/development/DS-License.html